scripts/testing/tolerance_test.py

```diff
@@ -73,6 +76,12 @@ def ensure_timesteps(baseline_curves, test_curves):
 
     baseline_timesteps = baseline_curves[timestep_name]
     test_timesteps = test_curves[timestep_name]
+    
+    # case if curve is a single value
+    if not is_list(baseline_curves[timestep_name]):
```

Comment:
Can you move this into a function that will always return a list regardless of what is in the json?

Something like:

```
 baseline_timesteps = get_list(baseline_curves, timestep_name)
```

Reply:
since in other places of the code there is a chance we need two names, i decided to create an `as_list(v)` function with only the list as an arg instead.

ex: `baseline_values = as_list(baseline_curves[field_name][stat_name])`

ive been testing the tolerance test by itself and i think i've fixed this issue in all the places i missed.